python/CMakeLists.txt

```
@@ -81,3 +81,7 @@ else()
 endif()
 
 install(TARGETS _basixcpp LIBRARY DESTINATION basix)
+
+# and the nanobind typing stubs.
+nanobind_add_stub(_basixcpp_stub MODULE _basixcpp OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/_basixcpp.pyi" MARKER_FILE "${CMAKE_CURRENT_BINARY_DIR}/py.typed" PYTHON_PATH ${CMAKE_CURRENT_BINARY_DIR} DEPENDS _basixcpp)
```

```suggestion
nanobind_add_stub(_basixcpp_stub MODULE _basixcpp OUTPUT $<TARGET_FILE_DIR:_basixcpp>/_basixcpp.pyi MARKER_FILE $<TARGET_FILE_DIR:_basixcpp>/py.typed PYTHON_PATH $<TARGET_FILE_DIR:_basixcpp> DEPENDS _basixcpp)
```

`TARGET_FILE_DIR ` should include the build specific dir.